test/Microsoft.TemplateSearch.TemplateDiscovery.IntegrationTests/TemplateDiscoveryTests.cs

```diff
@@ -46,7 +45,7 @@ public async Task CanRunDiscoveryTool()
             foreach (var cacheFilePath in cacheFilePaths)
             {
                 Assert.True(File.Exists(cacheFilePath));
-                new DotnetNew3Command(_log)
+                new DotnetCommand(_log, "new")
```

Comment:
Just curious - this will now not exercise local code changes form this repo - will it?

Reply:
We don't have exercisable code anymore, so this will run on SDK version installed. 
That's actually the good point - if we change anything in search cache, it won't be possible to get tested immediately. 
Likely we will need to go towards using unit tests instead, or open for other ideas.

Reply:
It feels that those test should rather be in sdk then.
Discovery should have only unit tests in templating repo